R/pin-read-write.R

```diff
@@ -64,6 +64,10 @@ pin_read <- function(board, name, version = NULL, hash = NULL, ...) {
 #'   use the default for `board`
 #' @param tags A character vector of tags for the pin; most important for
 #'   discoverability on shared boards.
+#' @param check_hash Check whether the pin contents are identical to the last
+#'   version if one exists (using the hash), and then **do not store** the pin
+#'   again. This argument does not check the pin metadata, only the pin
```

Comment:
This would actually be better because there is an internal function already called `check_hash()`. Take a look and see if you have feedback on the new name/docs.